02-Reproject-Raster-In-R.Rmd

```diff
@@ -11,8 +11,8 @@ category: spatio-temporal-workshop
 tags: [raster-ts-wrksp, raster]
 mainTag: raster-ts-wrksp
 description: "This lesson explains how to reproject a raster in R using the
-projectRaster() function in the raster package."
-code1: Reproject-Raster-In-R.R
+`projectRaster()` function in the raster package."
```

Comment:
It appeared to when I rendered on localhost.  However, since the .R files are in the main folder not the _code folder the button doesn't link properly.  However, the button does appear (I assume that is due to a template elsewhere, not just this one line).